### ISSUE TITLE
WCAG 3 for personal and small business websites

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/wcag3#388]
This issue is per request of @jspellman, as discussed on today's Silver call. The numbers in this post aren't real costs of any particular organisation, they're just examples.

Testing a site for WCAG is labor intensive. Depending on a variety of factors, it often takes at least a full day of work, often quite a bit more. For an organization of a budget of 100k doing an annual accessibility audit isn't a big deal. But if you have 8k to build a site, it is hard to justify spending 2k of it on an accessibility audit. That's not even fixing any accessibility problems, it's just to test for them.

The need for accessibility is universal. So WCAG 3 should be too. In my opinion the lowest conformance level of WCAG 3 should be one that is achievable by low-budget and no-budget websites. Ideally it is one that can be tested with minimal effort, without the involvement of an expert.

Having a lower entry level to WCAG 3, makes WCAG more inclusive. It may work better in countries where there is no legal presidency too, and it gives a starting point for organizations new to accessibility.

**Comment by alastc** [comment]
Hi @WilcoFiers,

> the lowest conformance level of WCAG 3 should be one that is achievable by low-budget and no-budget websites. Ideally it is one that can be tested with minimal effort, without the involvement of an expert.

I think there's a principle here: Should a level of conformance be set by the effort required? 

There was a similar proposal for automated testing being part of a first level ([survey](https://www.w3.org/wbs/35422/conformance-updates-test/results/#xq5), [minutes](https://www.w3.org/2026/05/11-ag-minutes.html#36d6)), but it was thought that: What is easy to test doesn't make it important to users.

The current draft has just one level of conformance, but multiple [tiers of reporting](https://www.w3.org/TR/2026/WD-wcag-3.0-20260910/#tiers-of-reporting). The tiers are based on user-impact rather than effort, but there is "a lower entry level". 

It's a slightly different approach than the one suggested, but does it fulfill the goal for you?
